README.md

```diff
@@ -1,10 +1,11 @@
-# ckb-next: RGB Driver for Linux
+# ckb-next: Corsair Driver for Linux
 
 <a target="_blank" href="https://web.libera.chat/?channels=#ckb-next"><img src="https://img.shields.io/badge/%23irc-libera.chat-blue.svg" height="20" alt="irc.libera.chat #ckb-next" /></a>
+[![Chat on Matrix](https://matrix.to/img/matrix-badge.svg)](https://matrix.to/#/#ckb-next:matrix.org)
 
-**ckb-next** is an open-source driver for Corsair keyboards and mice. It aims to bring the features of Corsair's proprietary CUE software to Linux operating systems. This project is currently a work in progress, but it already supports much of the same functionality, including full RGB animations. More features are coming soon. Testing and bug reports are appreciated!
+**ckb-next** is an open-source driver for Corsair devices. It aims to bring the features of Corsair's proprietary iCUE software to Linux operating systems. This project is currently a work in progress, but it already supports much of the same functionality, including full RGB animations. More features are coming soon. Testing and bug reports are appreciated!
 
-> __DISCLAIMER__: ckb-next is not an official Corsair product. It is licensed under the GNU General Public License (version 2) in the hope that it will be useful, but with NO WARRANTY of any kind.
+***Disclaimer:*** ckb-next is not an official Corsair product. It is not supported by, affiliated with, or endorsed by Corsair. It is licensed under the GNU General Public License (version 2) in the hope that it will be helpful, but with NO WARRANTY of any kind.
```

Comment:
The last sentence, especially the word `useful` is part of the GPL. https://github.com/ckb-next/ckb-next/blob/50cb44c655fb883bb60014c74de52fc73ffa8e42/LICENSE#L301-L302

If anything, it should be phrased more like it (the warranty part at the end).